query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/notUsingSchemaBase/NestedUpdateMutationInsideUpdateSpec.scala

```diff
@@ -1165,8 +1166,8 @@ class NestedUpdateMutationInsideUpdateSpec extends FlatSpec with Matchers with A
     server.queryThatMustFail(
       updateMutation,
       project,
-      errorCode = 3041,
-      errorContains = """The relation BottomToMiddle has no node for the model Middle connected to a Node for the model Bottom on your mutation path."""
+      errorCode = 0, // 3041,
+      errorContains = """Error occurred during query execution:\nInterpretationError(\"Error for binding \\'3\\': AssertionError(\\\"Expected a valid parent ID to be present for nested update to-one case.\\\")\")"""
```

Comment:
I think this should be an instance of the user facing `MissingRequiredValue` error, wdyt?